README.md

```diff
@@ -1 +1,3 @@
-# GitHubBot
\ No newline at end of file
+# GitHubBot
```

Comment:
Suggested code improvement:
```suggestion
```



# Github Bot

Github Bot is an awesome bot that automates and simplifies the process of creating and managing Pull Requests (PRs) on your Github repositories. It allows you to easily create and manage PRs and to quickly review existing PRs. It is a great way to keep your source code organized and updated.
```
```